cmd/restic/cmd_prune.go

```diff
@@ -75,241 +112,442 @@ func newProgressMax(show bool, max uint64, description string) *restic.Progress
 	return p
 }
 
-func runPrune(gopts GlobalOptions) error {
-	repo, err := OpenRepository(gopts)
+func runPrune(opts PruneOptions, gopts GlobalOptions) error {
+	err := verifyPruneFlags(opts)
 	if err != nil {
 		return err
 	}
 
-	lock, err := lockRepoExclusive(repo)
```

Comment:
Please add the exclusive lock back. Just noticed this when a backup run, that started in parallel to prune, blew up in my face. 🙄

Reply:
Oops, that was in fact not intended.. 
Don't remember when I removed it. After I added all flags also to `forget` I never used plain `prune` again, so this didn't show up to me..

Is fixed now.